src/utils.rs

```diff
@@ -18,27 +18,27 @@ pub fn commit_file<P: AsRef<Path>>(
 ) -> Result<(), Error> {
     
     let repo = try!(get_repo_from_path(repo_path));
-
-    let _ = repo.index()
+    let tree_oid = try!(repo.index()
```

Comment:
Okay, so I don't really like these changes. It seems like it got more complex. I think how you had it before was better. The only change I was looking for was that line 22 changed from a discarding assignment to a return.

I really dislike the code pattern where we have a bunch of stuff tryed! and then ended with an `Ok(())`. I think you should just get rid of the last try! and change it to a return with a `.map(|_| OK(()))`. What do you think?